design/overview.css

```diff
@@ -0,0 +1,100 @@
+/* ------- GRID SECTION --------- */ 
+
+.content-container {
+	 width: 90%;
+	 margin: 0px;
+}
+
+.grid-container {
```

Comment:
Grid is more powerful than just of columns, right? Could we use it for the entire page layout?

Reply:
Grid, in this case, allows for an indepent flex-box context per column, in which spacing can be set tos spread vertically. This gives the illusion of both horizontal and vertical spacing directions which I believe is impossible with just flex-boxes.

Reply:
So for the second question, I do not think we currently have any  other place in which grids are useful. If possible I prefer using exclusively flex-boxes as they are easier to comprehend.